pkgs/by-name/pi/pixivfe/package.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitea,
+  makeBinaryWrapper,
+}:
+
+# NOTE: Please do not forget to manually update these fields!
+let
+  revision = "19b09e7";
+  date = "2025.01.10";
+in
+buildGoModule rec {
+  pname = "pixivfe";
+  version = "2.11";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "VnPower";
+    repo = "PixivFE";
+    rev = "v${version}";
```

```suggestion
    tag = "v${version}";
```
you maybe need to rebase on master for this to work